perfmetrics/scripts/presubmit/print_results.py

```diff
@@ -12,30 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import sys
-# For fetching fio_metrics from fio
-sys.path.append("./perfmetrics/scripts/")
-from fio.fio_metrics import FioMetrics
+from prettytable import PrettyTable
 
-if __name__ == '__main__':
-  argv = sys.argv
-  if len(argv) != 2:
-    raise TypeError('Incorrect number of arguments.\n'
-                    'Usage: '
-                    'python3 show_results.py <fio output json filepath>')
+data = []
+for line in open("result.txt", "r").readlines():
+  data.append(line.strip())
 
-  # Fetching metrics from json file
-  fio_metrics_obj = FioMetrics()
-  data = fio_metrics_obj.get_metrics(argv[1])
+table = PrettyTable(
+    ["Branch", "File Size", "Read BW", "Write BW", "RandRead BW", "RandWrite BW"]
+)
 
-  # Fetching results in result.txt file
-  file = open("result.txt", "a")
-  # Iterating through data
-  for d in data :
-    # Print filesize only once
-    if d['params']['rw'] == "read":
-      file.write(str.format("\n"+"Filesize: "+ str(round(d["params"]["filesize_kb"]/1024.0,3)) + "MiB" + "\n"))
+for i in range(0, 15, 5):
```

Comment:
Please create variables for these numbers:
E.g.
data_dimension = 5 
data_set_split_index = 15 # data at [0, data_set_split_index) belong to Master branch and [data_set_split_index, size) belongs to PR branch.

Reply:
fixed